src/data/accessibility.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+import { Platform } from "react-native";
+
+const checkboxAccessibilityLabel = (label: string, checked: boolean) =>
+  Platform.select({
```

Comment:
Do we want to use 2 separate files (accessibility.ios.js and accessibility.android.js) instead of `Platform.select`? I think this is cleaner because then iOS does not need to load the Android code and vice versa.

Reply:
If you think that is best? I'm happy to